trivy-task/index.ts

```diff
@@ -15,6 +15,9 @@ async function run() {
 
     let scanPath = task.getInput("path", false)
     let image = task.getInput("image", false)
+    let loginDockerConfig = task.getBoolInput("loginDockerConfig", false)
+    let ignoreUnfixed = task.getBoolInput("ignoreUnfixed", false)
+    let severities = task.getInput("severities", false)
```

Comment:
```suggestion
    let severities = task.getInput("severities", false) ?? ""
```

Reply:
hi @ferronsw, sorry for the back and forth but we have a test failing that is ensuring that the severities variable is a string. unless we add this logic, theres a chance the string passed as a parameter on lines 50/52 could also be undefined (for the purpose of the build). 

Error: index.ts(50,59): error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.
Error: index.ts(52,59): error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.

Reply:
No problem.